fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -60,18 +60,44 @@
   </xsl:template>
 
 
+  <!-- the following template creates an _ms field for thesis advisors -->
+  <xsl:template match="mods:mods/mods:name[(mods:role/mods:roleTerm='Thesis advisor') or 
+    (mods:role/mods:roleTerm='thesis advisor')]" mode="utk_ir_MODS">
+
+    <xsl:for-each>
```

Comment:
I'm pretty sure this stylsheet won't compile. I don't know if you've had a chance to test this locally yet, but `xsl:for-each` has a **mandatory** select attribute. 

Edit: additionally, I'm pretty sure this won't quite work the way the template is articulated. Take a look at the [`xsl:for-each` for mods:abstract(s)](https://github.com/utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config/blob/institutional_repository/fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt#L85-L92).